src/scripts/npm/updateNativeSdk.sh

```diff
@@ -62,23 +62,11 @@ update_ios() {
 }
 
 update_and() {
-  # properties
```

Comment:
Was this all removed due to the switch to using the aar?

Reply:
Yes.